webapp/locales/de.json

```
@@ -143,27 +148,43 @@
             "successDelete": "Social-Media gelöscht. Profil aktualisiert!"
         },
         "muted-users": {
-            "name": "Stummgeschaltete Benutzer",
-            "explanation": {
-                "intro": "Wenn ein anderer Benutzer von dir stummgeschaltet wurde, dann passiert folgendes:",
-                "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der stummgeschalteten Person mehr auf.",
-                "search": "Die Beiträge von stummgeschalteten Personen verschwinden aus deinen Suchergebnissen."
-            },
-            "columns": {
-                "name": "Name",
-                "slug": "Alias",
-                "unmute": "Entsperren"
-            },
-            "empty": "Bislang hast du niemanden stummgeschaltet.",
-            "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü stummschalten.",
-            "mute": "Stumm schalten",
-            "unmute": "Stummschaltung aufheben",
-            "unmuted": "{name} ist nicht mehr stummgeschaltet"
-        },
-        "privacy": {
-            "name": "Privatsphäre",
-            "make-shouts-public": "Teile von mir empfohlene Artikel öffentlich auf meinem Profil",
-            "success-update": "Privatsphäre-Einstellungen gespeichert"
+          "name": "Stummgeschaltete Benutzer",
+          "explanation": {
+            "intro": "Wenn ein anderer Benutzer von dir stummgeschaltet wurde, dann passiert folgendes:",
+            "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der stummgeschalteten Person mehr auf.",
+            "search": "Die Beiträge von stummgeschalteten Personen verschwinden aus deinen Suchergebnissen."
+          },
+          "columns": {
+            "name": "Name",
+            "slug": "Alias",
+            "unmute": "Entsperren"
+          },
+          "empty": "Bislang hast du niemanden stummgeschaltet.",
+          "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü stummschalten.",
+          "mute": "Stumm schalten",
+          "unmute": "Stummschaltung aufheben",
+          "unmuted": "{name} ist nicht mehr stummgeschaltet"
+        },
+        "blocked-users": {
+          "name": "Blocked users",
+          "explanation": {
+            "intro": "Wenn ein anderer Benutzer von dir blockiert wurde, dann passiert folgendes:",
+            "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der blockierten Person mehr auf.",
```

```suggestion
            "your-perspective": "Du kannst keine Beiträge der blockierten Person mehr kommentieren.",
```
Translates to: "You cannot comment on posts of the blocked person"